main.go

```diff
@@ -687,9 +691,14 @@ The validate command expects a configuration file as its only argument. Local fi
 			config, err := agentConfig.Scan(collector.Directories(constants.GetYipConfigDirs()...), collector.NoLogs)
 			config.Strict = c.Bool("strict")
 
+			if len(c.StringSlice("override-cloud-init-paths")) > 0 {
```

Comment:
should this come after the extend?

Reply:
I thought at it, but then would be more confusing : you could specify both but only one of the two would take effect.

Reply:
mmm but this is the collector.. so things could come back from config :facepalm: